src/api/customer/customer-deserializer.ts

```diff
@@ -9,6 +9,12 @@ const createCustomerDeserializer: CustomerCreationRequestHandler  = (req, res, n
     next();
 }
 
+const findCustomersDeserializer: CustomersFindRequestHandler  = (req, res, next) => {
+    res.locals.customerToFind = {uuid: req.params.uuid}
```

Comment:
Poderia simplificar:
```suggestion
    res.locals.customerToFind = req.params;
```